x/mongo/driver/topology/pool_test.go

```diff
@@ -23,6 +23,22 @@ func TestPool(t *testing.T) {
 	t.Run("newPool", func(t *testing.T) {
 		t.Parallel()
 
+		t.Run("minPoolSize should not exceed maxPoolSize", func(t *testing.T) {
+			t.Parallel()
```

Comment:
Do subtests of tests marked with `t.Parallel()` also need `t.Parallel()`?

Reply:
We need `Parallel()` in every subtest, basically meaning `Parallel()` is paired with `Run()` to run the tests in parallel, in my understanding. However, I agree the `Parallel()` calls are not necessary for the subtests under "newPool" considering they are lightweight.

Reply:
Sounds good. Whether or not the `newPool` tests _need_ to be `Parallel()`, I don't have a strong opinion. Unless I'm mistaken, I don't think our CI yet runs tests in parallel. Parallelizing the test suite may be done with GODRIVER-2093.

Reply:
Awesome; thanks so much for that explanation.